web/client/components/map/openlayers/Map.jsx

```diff
@@ -439,7 +443,7 @@ class OpenlayersMap extends React.Component {
     updateMapInfoState = () => {
         let view = this.map.getView();
         let tempCenter = view.getCenter();
-        let projectionExtent = view.getProjection().getExtent() || getExtentForProjection(this.props.projection);
+        let projectionExtent = view.getProjection().getExtent() || msGetProjection(this.props.projection).extent;
```

Comment:
This is used twice. IDK exactly why OL extent may be null (and if it should have priority or not).
You can create a function getExtent(this.map, this.props) to centralize and help future refactors and/or changes.